atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -389,8 +389,13 @@ private void loadWithTs(TableReference tableRef,
                             ThreadSafeResultVisitor v,
                             ConsistencyLevel consistency) throws Exception {
         List<Callable<Void>> tasks = Lists.newArrayList();
-        for (Map.Entry<InetSocketAddress, List<Cell>> hostAndCells : partitionByHost(cells,
-                                                                               Cells.getRowFunction()).entrySet()) {
+        Map<InetSocketAddress, List<Cell>> hostsAndCells =  partitionByHost(cells, Cells.getRowFunction());
+        int i = 1;
+        int size = hostsAndCells.keySet().size();
+        for (Map.Entry<InetSocketAddress, List<Cell>> hostAndCells : hostsAndCells.entrySet()) {
+            log.trace("Making request {} of {} for a loadWithTs call.  It is against host {} and table {}",
+                    i, size, hostAndCells.getKey().getHostName(), tableRef.toString());
```

Comment:
We probably want to use getHostString() to avoid DNS lookup that getHostName does, see https://docs.oracle.com/javase/7/docs/api/java/net/InetSocketAddress.html#getHostName() . Might also be worth wrapping this in a `if (log.isTraceEnabled()) {`